app/controllers/bus_stops_controller.rb

```diff
@@ -4,9 +4,11 @@ class BusStopsController < ApplicationController
   before_action :set_fields_for_stop, only: %i[update edit]
 
   def autocomplete
-    stops = BusStop.where 'lower(name) like ?',
-                          "%#{params.require(:term)}%"
-    render json: stops.map(&:name_with_id).sort
+    stops = BusStop.where 'lower(name) like ?', "%#{params.require(:term)}%"
+    jsondata = stops.map { |stop| {label: stop.name_with_id,
```

Comment:
use `do/end` in place of `{}` for multi-line blocks (run rubocop to see what I mean)